plugins/woocommerce/tests/e2e-pw/.eslintrc.js

```diff
@@ -5,5 +5,6 @@ module.exports = {
 		'no-console': 'off',
 		'jest/no-test-callback': 'off',
 		'jest/no-disabled-tests': 'off',
+		'jest/valid-expect': 'off',
```

Comment:
Is this intentional?  It looked like each test had a valid assertion, but maybe I missed something.

Reply:
Yes, it's intentional. In one of the spec files I touched there are some custom expect messages, which are a Playwright test feature that Jest doesn't have and like. The Jest eslint rules should be disabled anyway, because we're using the `playwright/recommended` ones, but there is not way (I know of) to turn off all the rules of a plugin.

Reply:
Ah, thanks.  I didn't know that the linting objected to custom rules.  I thought it was just "dumb" and looked for at least one `expect` per test.  Learned something new!